### PR TITLE
Converted the project to Capital Framework v3

### PR DESCRIPTION
**Pull request by jimmynotjim**
Converted the project to Capital Framework v3

## Removals

- Removes normalize and normalize-legacy from main less file because CF already includes it.
- Removes old branded list mixin (was causing compile errors).

## Changes

- Moves from pulling CF from Bower to NPM (CF Tables is temp on bower until it's released on the main CF repo)

## Testing

- run `setup.sh` and ensure the project builds

## Review

- @KimberlyMunoz 
- @anselmbradford 
- @sebworks 
- @contolini 

## Notes

- We should consider moving the rest of the packages on Bower to NPM if possible

## Checklist

* [ ] Changes are limited to a single goal (no scope creep)
* [ ] Code can be automatically merged (no conflicts)
* [ ] Code follows the standards laid out in the [front end playbook](https://github.com/cfpb/front-end)
* [ ] Passes all existing automated tests
* [ ] New functions include new tests
* [ ] New functions are documented (with a description, list of inputs, and expected output)
* [ ] Placeholder code is flagged
* [ ] Visually tested in supported browsers and devices 
* [ ] Project documentation has been updated (including the "Unreleased" section of the CHANGELOG)

### CODE REVIEW

File: gulp/config.js [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 var fs = require( 'fs' );
 var paths = require( '../config/environment' ).paths;
+var glob = require( 'glob-all' );
```
Comment (sebworks) [inline gulp/config.js]: Should this be called `globAll` instead, so devs don't think its just `glob`?

Comment (jimmynotjim) [inline gulp/config.js]: Good call, fixed in ac9b3ab

**Comment by sebworks** [comment]
:+1:

**Comment by KimberlyMunoz** [comment]
image

:scream:

**Comment by KimberlyMunoz** [comment]
There seems to be 404s on the cf-icons fonts.

**Comment by contolini** [comment]
re: icons

jumping out window

**Comment by jimmynotjim** [comment]
Icons are fixed, thanks for catching that.

**Comment by KimberlyMunoz** [comment]
:+1:

**Pull request merged**